services/user_interface/templates/swipe.html

```diff
@@ -93,15 +116,28 @@
     <div class="row">
         <div class="col-md-12">
             <h1 class="text-center">
-                Active Manifesto
+                Active Manifesto - Political Statement Tagger
             </h1>
         </div>
     </div>
     <div class="row">
         <div class="col-md-12">
             <h2>
-                Political Samples
+                Instructions
```

Comment:
fuer die fertige app waers cool, wenn nich so viel text da stuende, wegen platz, also dann spaeter koennten wir die instructions weglassen, glaub ich.

Reply:
Gute Idee, ich pack den Text einfach in die graue Box fuers swipen. Die Instructions werden dann vor jeder labelling runde angezeigt, ok?

Reply:
super!